modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,156 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
+
+:_content-type: PROCEDURE
+[id='replacing-a-bare-metal-control-plane-node_{context}']
+= Replacing a bare metal control plane node
+
+Use the following procedure to replace an installer-provisioned {product-title} cluster.
+
+[IMPORTANT]
+====
+When the replacement BareMetalHost is created from a new physical machine or an incomplete prior YAML file configuration, the `externallyProvisioned:` field in the configuration file must be set to `false` to allow the cluster to provision the host. 
+
+Existing control plane BareMetalHosts may have the externallyProvisioned flag set to true if they were provisioned by the OpenShift Installer.
+====
+
+.Procedure
+
+. Create the baremetalhost and ipmi secret:
+
+[source,terminal]
+----
+$ cat <<EOF | oc apply -f -
+----
+
+.Example output
+[source,terminal]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: master-3-bmc-secret
+  namespace: openshift-machine-api
+type: Opaque
+data:
+  username: _<username>_
+  password: _<password>_
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: master-3
+  namespace: openshift-machine-api
+spec:
+  externallyProvisioned: false
+  online: true
+  bootMACAddress: aa:aa:aa:aa:aa:03
+  bmc:
+	address: ipmi://172.22.0.1:6233
+	credentialsName: master-3-bmc-secret
+EOF
+----
+
+. Once the inspection finishes, the bare metal host will be created and 'ready' to be provisioned:
```

Comment:
The `ready` state was renamed to `available` in ~4.9 (don't quote me on the exact release)